pkg/plugins/runtime/k8s/webhooks/validation.go

```diff
@@ -2,26 +2,52 @@ package webhooks
 
 import (
 	"context"
+	"fmt"
 	"net/http"
 
 	"k8s.io/api/admission/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kube_runtime "k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
+	"github.com/kumahq/kuma/pkg/config/mode"
+	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
 	core_model "github.com/kumahq/kuma/pkg/core/resources/model"
 	core_registry "github.com/kumahq/kuma/pkg/core/resources/registry"
 	"github.com/kumahq/kuma/pkg/core/validators"
+	kds_global "github.com/kumahq/kuma/pkg/kds/global"
+	kds_remote "github.com/kumahq/kuma/pkg/kds/remote"
+	common_k8s "github.com/kumahq/kuma/pkg/plugins/common/k8s"
 	k8s_resources "github.com/kumahq/kuma/pkg/plugins/resources/k8s"
 	k8s_model "github.com/kumahq/kuma/pkg/plugins/resources/k8s/native/pkg/model"
 	k8s_registry "github.com/kumahq/kuma/pkg/plugins/resources/k8s/native/pkg/registry"
 )
 
-func NewValidatingWebhook(converter k8s_resources.Converter, coreRegistry core_registry.TypeRegistry, k8sRegistry k8s_registry.TypeRegistry) AdmissionValidator {
+var multiclusterSyncValidationResp = v1beta1.AdmissionResponse{
```

Comment:
Probably better to have it as a function that returns a new object, to avoid calling `DeepCopy` every time?